specs/pvs/corla.pvs

```diff
@@ -781,17 +828,18 @@ END cryptography
 
 %%% \subsection{Randomness}
 
-%%% High quality randomness is key to the legitimacy of any
-%%% risk-limiting audit.
+%%% Choosing a random sample of ballots is key to the legitimacy of any
+%%% risk-limiting audit. Legitimacy depends on a high-quality process for
+%%% the random samples. The Colorado RLA Tool will produce a random order 
+%%% of all the ballots once and for all; then random samples can be created 
```

Comment:
That would work, but I'm thinking it would make it harder for the public to verify the selection, since now they have to include all manifests when they do so, in some canonical order among counties, and then do their own winnowing of the overall list to generate what is selected for a given county. They couldn't e.g. just feed a single county manifest (with a bit of tweaking) into Philip's tool and confirm the sample that way. I'll have to think about this....